apps/src/tutorialExplorer/search.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import FilterGroupContainer from './filterGroupContainer';
+import SearchBar from '../templates/SearchBar';
```

Comment:
I think we should make some aesthetic improvements to this component.  For starters: the X only needs to be shown/enabled when there is content to clear, otherwise it looks like it might dismiss the search box; the X should be vertically centered; the X should have `cursor: pointer`.

Reply:
I made a few small adjustments per your suggestions, also shows fix for the "oceans" problem you discovered, still working on the squirrelly scrolling: 

https://user-images.githubusercontent.com/12300669/133487750-739d66be-b7d7-46a1-b609-9ae21ccf8fdc.mov